testapp/tests-jelix/jelix/forms/jforms_compilerTest.php

```diff
@@ -13,7 +13,7 @@
 
 require_once(JELIX_LIB_PATH.'forms/jFormsCompiler.class.php');
 require_once(JELIX_LIB_PATH.'forms/jFormsCompiler_jf_1_0.class.php');
-require_once(JELIX_LIB_PATH.'forms/jFormsControl.class.php');
+//require_once(JELIX_LIB_PATH.'forms/jFormsControl.class.php');
```

Comment:
remove this line directly. And probably all others requires about `jForms*` classes

Reply:
done, forms/jFormsDataSource.class.php needs to split each class in dedicated file.
i'm on it..